studio/components/interfaces/BillingV2/Usage/Usage.tsx

```diff
@@ -0,0 +1,166 @@
+import { useProjectUsageQuery } from 'data/usage/project-usage-query'
+import Link from 'next/link'
+import { useRef, useState } from 'react'
+import { Button, IconAlertCircle, IconCheckCircle, IconLoader, Listbox, Toggle } from 'ui'
+import Activity from './Activity'
+import Bandwidth from './Bandwidth'
+import Infrastructure from './Infrastructure'
+import SizeAndCounts from './SizeAndCounts'
+import { USAGE_CATEGORIES, USAGE_STATUS } from './Usage.constants'
+import { getUsageStatus } from './Usage.utils'
+
+const Usage = () => {
+  const { ref } = useParams()
+  const [selectedProjectRef, setSelectedProjectRef] = useState<string>(ref as string)
+
+  const infrastructureRef = useRef<HTMLDivElement>(null)
+  const bandwidthRef = useRef<HTMLDivElement>(null)
+  const sizeAndCountsRef = useRef<HTMLDivElement>(null)
+  const activityRef = useRef<HTMLDivElement>(null)
+
+  const { data: projects, isLoading } = useProjectsQuery()
+  const { data: usage } = useProjectUsageQuery({ projectRef: ref })
+  const { data: subscription, isLoading: isLoadingSubscription } = useProjectSubscriptionQuery({
+    projectRef: selectedProjectRef,
+  })
+
+  const scrollTo = (id: 'infra' | 'bandwidth' | 'sizeCount' | 'activity') => {
+    switch (id) {
+      case 'infra':
+        if (infrastructureRef.current)
+          infrastructureRef.current.scrollIntoView({ behavior: 'smooth' })
+        break
+      case 'bandwidth':
+        if (bandwidthRef.current) bandwidthRef.current.scrollIntoView({ behavior: 'smooth' })
+        break
+      case 'sizeCount':
+        if (sizeAndCountsRef.current)
+          sizeAndCountsRef.current.scrollIntoView({ behavior: 'smooth' })
+        break
+      case 'activity':
+        if (activityRef.current) activityRef.current.scrollIntoView({ behavior: 'smooth' })
+        break
+    }
+  }
+
+  return (
+    <>
+      <div className="border-b">
+        <div className="1xl:px-28 mx-auto flex flex-col px-5 lg:px-16 2xl:px-32 pt-6 space-y-4">
+          <h3 className="text-scale-1200 text-xl">Usage</h3>
+          <div className="flex items-center justify-between">
+            <div className="flex items-center space-x-4">
+              <Listbox
+                disabled
+                size="small"
+                id="billingCycle"
+                name="billingCycle"
+                value={'current'}
+                className="!w-[200px]"
+                onChange={() => {}}
+              >
+                <Listbox.Option label="Current billing cycle" value="current">
+                  Current billing cycle
+                </Listbox.Option>
+                <Listbox.Option label="Previous billing cycle" value="previous">
+                  Previous billing cycle
+                </Listbox.Option>
+              </Listbox>
+              {isLoading ? (
+                <ShimmeringLoader className="w-[200px]" />
+              ) : (
+                <Listbox
```

Comment:
As this is on the project page, I'd suggest we also disable the dropdown for now and only show the current project

Reply:
Especially for org level billing, it will become more interesting, but for now, I'd disable the dropdown for project selection

Reply:
yea hide it. spoke to @joshenlim about that just now.